.github/workflows/release_staging.yml

```diff
@@ -0,0 +1,77 @@
+name: Deriv App
```

Comment:
Can we have a meaningful name?

Reply:
answered here
https://github.com/binary-com/deriv-app/pull/10016#discussion_r1322315429